version

```diff
@@ -0,0 +1 @@
+2.0
```

Comment:
@sbasile-ch from the tags i could see the last higher major minor tag was 9.0.8 so here major version 2 and minor version 0 might fail in concourse during release job so its better to keep the version as 10.0

Reply:
Thanks @hselvakumar it does seem we kept the forked-from tags (2015) but then we numbered differently our releases (2017)
Updated.
<img width="317" alt="Screenshot 2023-06-02 at 09 35 05" src="https://github.com/companieshouse/flyingsaucer/assets/12054005/c2eea674-e53e-49d5-a521-01b00f7c751f">